### PR TITLE
repo: Release `1.24.0`

### PR DESCRIPTION
**Pull request by wrowe**
Signed-off-by: William A Rowe Jr <wrowe@vmware.com>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
strange - not sure why format check failed

/retest

**Review by phlax — DISMISSED** [review]
lgtm, thanks @wrowe

File: docs/versions.yaml [hunk 1]
```diff
@@ -13,6 +13,6 @@
 "1.17": 1.17.4
 "1.18": 1.18.4
 "1.19": 1.19.5
-"1.20": 1.20.6
-"1.21": 1.21.4
-"1.22": 1.22.2
+"1.20": 1.20.7
+"1.21": 1.21.5
+"1.22": 1.22.5
```
Comment (wrowe) [inline docs/versions.yaml:18]: @phlax One question; why is 1.23.2 missing here?

Comment (moderation) [reply docs/versions.yaml:18]: The website repo was never updated for `1.23.1` and hasn't been updated for `1.23.2`

https://github.com/envoyproxy/envoy-website/blob/main/_config.yml#L23

Comment (phlax) [reply docs/versions.yaml:18]: yep - i must get round to automating that version number as it is often overlooked

in this case we can probably just jump straight to 1.24.0 once this lands

Comment (phlax) [reply docs/versions.yaml:18]: > @phlax One question; why is 1.23.2 missing here?

good question - it doesnt seem to have a 1.23.x entry at all - ill check further

Comment (phlax) [reply docs/versions.yaml:18]: not sure why it didnt include first time around - but running it locally - it does seem to add the expected version

```diff
diff --git a/docs/versions.yaml b/docs/versions.yaml
index 2335d2019d..202855aa27 100644
--- a/docs/versions.yaml
+++ b/docs/versions.yaml
@@ -13,6 +13,7 @@
 "1.17": 1.17.4
 "1.18": 1.18.4
 "1.19": 1.19.5
-"1.20": 1.20.6
-"1.21": 1.21.4
-"1.22": 1.22.2
+"1.20": 1.20.7
+"1.21": 1.21.5
+"1.22": 1.22.5
+"1.23": 1.23.2
```

the relevant `objects.inv` is also added

can you try removing the commit and running the tool again

**Comment by wrowe** [comment]
Replaced with https://github.com/envoyproxy/envoy/pull/23561

**Pull request closed without merging**